packages/agent/conf/demos/energy-shr/oid4vci_metadata/energyshr.json

```diff
@@ -59,40 +59,40 @@
           }
         ],
         "order": [
-          "Voornaam",
-          "Achternaam",
+          "firstName",
+          "lastName",
           "email",
           "type"
         ],
         "credentialSubject": {
-          "Voornaam": {
+          "firstName": {
             "value_type": "string",
             "display": [
               {
-                "name": "FirstName"
+                "name": "First Name"
```

Comment:
what should be the default locale here? dutch or english?

Reply:
see prev answer